app/templates/views/dashboard/dashboard.html

```diff
@@ -97,7 +110,7 @@ <h2 class="margin-top-4 margin-bottom-1">Recent Batches</h2>
       </table>
     </div>
 
-    <h2 class="margin-top-4 margin-bottom-1">Message count</h2>
+    {#  <h2 class="margin-top-4 margin-bottom-1">Message count</h2>
```

Comment:
Could we delete these if we don't need it anymore rather than commenting them out?